test/metabase/test/data/crate.clj

```diff
@@ -0,0 +1,75 @@
+(ns metabase.test.data.crate
+  "Code for creating / destroying a Crate database from a `DatabaseDefinition`."
+  (:require [environ.core :refer [env]]
+            metabase.driver.crate
+            (metabase.test.data [generic-sql :as generic]
+                                [interface :as i])
+            [metabase.driver.generic-sql :as sql]
+            [clojure.java.jdbc :as jdbc]
+            [metabase.util :as u]
+            [clojure.string :as s])
+  (:import metabase.driver.crate.CrateDriver))
+
+(def ^:private ^:const field-base-type->sql-type
+  {:BigIntegerField "long"
+   :BooleanField    "boolean"
+   :CharField       "string"
+   :DateField       "timestamp"
+   :DateTimeField   "timestamp"
+   :DecimalField    "integer"
+   :FloatField      "float"
+   :IntegerField    "integer"
+   :TextField       "string"
+   :TimeField       "timestamp"})
+
+
+(defn- timestamp->CrateDateTime
+  [value]
+  (if (= (instance? java.sql.Timestamp value) true)
+    (.getTime (u/->Timestamp value))
+    (if (= (and (= (instance? clojure.lang.PersistentArrayMap value) true) (contains? value :korma.sql.utils/generated)) true)
+      (+ (read-string (s/replace (:korma.sql.utils/generated value) #"CURRENT_TIMESTAMP \+" "")) (.getTime (u/new-sql-timestamp)))
```

Comment:
You should be using `(korma.core/raw value)` instead of constructing that function's results yourself like `{:korma.sql.utils value}`